Makefile

```diff
@@ -36,12 +36,16 @@ build:
 	GO111MODULE=on go build -v -o .
 
 .PHONY: build-all
-build-all: go-generate vendor build-windows-amd64 build-linux-amd64 build-linux-arm64 build-darwin-amd64 build-darwin-arm64
+build-all: go-generate vendor build-windows-amd64 build-windows-386 build-linux-amd64 build-linux-arm64 build-darwin-amd64 build-darwin-arm64
 
 .PHONY: build-windows-amd64
 build-windows-amd64:
 	GOOS=windows GOARCH=amd64 go build -ldflags "-X github.com/Azure/draft/cmd.VERSION=${DRAFT_VERSION}" -v -o ./bin/draft-windows-amd64.exe
 
+.PHONY: build-windows-386
```

Comment:
hi Thank you. Just curious are there still win32 users? ex: it old 32-bit windows which is not supported.

Reply:
my understanding of [this](https://github.com/Azure/aks-devx-tools/actions/runs/3715528818/jobs/6300805074#step:6:130) failure is that the github action runner needs a 32bit windows binary since it refers to "ia32" as the host architecture, which i recognize is weird since the runner image says it's amd64, but it could be a compatibility mode situation where we run the vscode debugger in 32bit mode

Reply:
@hsubramanianaks good point you may be correct, since I was able to force the runner to use the amd64 build on its ia32 architecture, so I'll just add an [architecture remap](https://github.com/Azure/aks-devx-tools/pull/80/commits/77589e11df5bc5adabe708340070b3016dc58d55) for this case